pkg/lib/machine-info.es6

```diff
@@ -191,3 +191,46 @@ export function udev_info(address) {
     }
     return pr;
 }
+
+const memoryRE = /^(\w+[ \w+]*):\s(.*)/;
```

Comment:
IMHO this can be simplified to `^([ \w]+): (.*)/`. I. e. we expect the key name to consist of "word" and space characters.